central/pruning/pruning.go

```diff
@@ -430,10 +438,18 @@ func (g *garbageCollectorImpl) removeOrphanedProcessBaselines(deployments set.Fr
 // removeOrphanedPLOPs: cleans up ProcessListeningOnPort objects that are expired
 // or have a PodUid and belong to a deployment or pod that does not exist.
 func (g *garbageCollectorImpl) removeOrphanedPLOPs() {
-	prunedCount := postgres.PruneOrphanedPLOPs(pruningCtx, g.postgres, orphanWindow)
+	prunedCount := g.plops.PruneOrphanedPLOPs(pruningCtx, orphanWindow)
 
 	log.Infof("[PLOP pruning] Found %d orphaned process listening on port objects",
 		prunedCount)
+
+	prunedCount, err := g.plops.RemovePLOPsWithoutProcessIndicatorOrProcessInfo(pruningCtx)
+
+	if err != nil {
+		log.Error(errors.Wrap(err, "error removing PLOPs with no matching process indicator or process information"))
```

Comment:
No need to wrap in this case, can just log.Errorf("error removing PLOPs with no matching process indicator or process information: %v", err)

Reply:
wrapping is useful for tracing the error back but in this case, since we're logging, it's not needed

Reply:
Done